src/components/reusable/table/table-header.ts

```diff
@@ -86,10 +137,28 @@ export class TableHeader extends LitElement {
       'sr-only': this.visiblyHidden,
     };
 
+    const role = this.sortable ? 'button' : undefined;
+    const arialSort = this.sortable ? this.sortDirection : undefined;
+    const ariaLabel =
+      this.sortable && this.headerLabel
+        ? `Sort by ${this.headerLabel}`
+        : undefined;
+
     return html`
       <div
         class="container"
         @click=${this.sortable ? () => this.toggleSortDirection() : undefined}
+        role=${role}
```

Comment:
You can use Lit's ifDefined decorator here: https://lit.dev/docs/api/directives/#ifDefined